### PR TITLE
Additional gantt features

### PR DESCRIPTION
**Pull request by dmo-odoo**

### CODE REVIEW

**Comment by dmo-odoo** [comment]
Enterprise PR: https://github.com/odoo/enterprise/pull/3669

File: doc/reference/views.rst [hunk 1]
```diff
@@ -1350,6 +1355,9 @@ take the following attributes:
   * ``week``
   * ``month``
   * ``year``
+``scales``
+  coma-separated list of allowed scales for this view. By default, all scales
```
Comment (aab-odoo) [inline doc/reference/views.rst]: comma

**Comment by aab-odoo** [comment]
robodoo rebase-ff

**Comment by aab-odoo** [comment]
robodoo r+

**Pull request closed without merging**